dags/xlml/solutionsteam_tf_nightly_supported.py

```diff
@@ -53,15 +54,30 @@
       time_out_in_min=60,
   ).run()
 
-  tf_resnet_v4_32 = tf_config.get_tf_resnet_config(
-      tpu_version="4",
-      tpu_cores=32,
-      tpu_zone=vm_resource.Zone.US_CENTRAL2_B.value,
+  tf_resnet_v5e_4 = tf_config.get_tf_resnet_config(
+      project_name=vm_resource.Project.TPU_PROD_ENV_AUTOMATED.value,
+      tpu_version="5litepod",
+      tpu_cores=4,
```

Comment:
nit for a future PR - we might want to switch to using topologies (e.g. 2x2) or chips specifically since our naming convention is migrating away from cores

Reply:
(this can be a future PR)

Reply:
Sounds good. I recall the topologies is used starting from v4. Does it also apply to v2 and v3? But we definitely need to add this term to provide more possibilities.

Reply:
Discussed offline.